### PR TITLE
MM-37647 - modify branding svg images for cloud system console

### PR DESCRIPTION
**Pull request by pablovelezvidal**
#### Summary
This PR updates the SVG images present in the cloud billing and payment section of the system console, to the new branding UX.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-37647

#### Screenshots
Captura de pantalla 2021-08-11 a las 13 09 20
Captura de pantalla 2021-08-11 a las 13 16 05
Captura de pantalla 2021-08-11 a las 13 21 23
Captura de pantalla 2021-08-11 a las 14 18 55
Captura de pantalla 2021-08-11 a las 14 20 45

#### Release Note
```release-note
Updated the images present in the cloud billing and payment section to the new branding
```

### CODE REVIEW

**Comment by matthewbirtch** [comment]
Looks good from the screenshots @pablovelezvidal. However, I just spun up a test server (Setup Cloud + CWS Test Server) and the new illustrations don't show in the test server. How can I review?

**Comment by pablovelezvidal** [comment]
> Looks good from the screenshots @pablovelezvidal. However, I just spun up a test server (Setup Cloud + CWS Test Server) and the new illustrations don't show in the test server. How can I review?

There is an issue with the spinwick server, it is not taking the code from the branch, but from another old commit. I will find how to fix it and let you know when it is ready. @matthewbirtch

File: components/admin_console/billing/billing_history.tsx [hunk 1]
```diff
@@ -28,9 +29,9 @@ const PAGE_LENGTH = 4;
 
 const noBillingHistorySection = (
     <div className='BillingHistory__noHistory'>
-        <img
-            className='BillingHistory__noHistory-graphic'
```
Comment (neallred) [inline components/admin_console/billing/billing_history.tsx:32]: Nit: Can we also remove the class in `components/admin_console/billing/billing_history.scss`

File: components/admin_console/billing/billing_subscriptions/billing_subscriptions.tsx [hunk 2]
```diff
@@ -9,10 +9,8 @@ import {trackEvent} from 'actions/telemetry_actions';
 import BlockableLink from 'components/admin_console/blockable_link';
 import AlertBanner from 'components/alert_banner';
 
-import privateCloudImage from 'images/private-cloud-image.svg';
-import freeTrialPrivateCloudImage from 'images/free-trial-private-cloud-image.svg';
```
Comment (neallred) [inline components/admin_console/billing/billing_subscriptions/billing_subscriptions.tsx:13]: Nit: Can we remove these svg files? It doesn't look like they're used elsewhere in webapp.

File: components/admin_console/billing/billing_summary/billing_summary.tsx [hunk 3]
```diff
@@ -10,19 +10,18 @@ import {BillingSchemes, CloudLinks, TrialPeriodDays} from 'utils/constants';
...
+import ResetPasswordSvg from 'components/common/svg_images_components/reset_password.svg';
 
 import {trackEvent} from 'actions/telemetry_actions';
 
 import {Client4} from 'mattermost-redux/client';
 
 export const noBillingHistory = (
     <div className='BillingSummary__noBillingHistory'>
-        <img
-            className='BillingSummary__noBillingHistory-graphic'
```
Comment (neallred) [inline components/admin_console/billing/billing_summary/billing_summary.tsx:24]: Nit: can we remove the class definition in `components/admin_console/billing/billing_summary/billing_summary.scss`?

File: components/admin_console/billing/billing_summary/billing_summary.tsx [hunk 4]
```diff
@@ -10,19 +10,18 @@ import {BillingSchemes, CloudLinks, TrialPeriodDays} from 'utils/constants';
 import FormattedMarkdownMessage from 'components/formatted_markdown_message';
 import BlockableLink from 'components/admin_console/blockable_link';
 import OverlayTrigger from 'components/overlay_trigger';
-
-import noBillingHistoryGraphic from 'images/no_billing_history_graphic.svg';
```
Comment (neallred) [inline components/admin_console/billing/billing_summary/billing_summary.tsx:14]: Nit: can we remove both these images? I don't see them used elsewhere.

File: components/admin_console/billing/company_info_display.tsx [hunk 5]
```diff
@@ -8,7 +8,8 @@ import {useSelector} from 'react-redux';
 import {trackEvent} from 'actions/telemetry_actions';
 import BlockableLink from 'components/admin_console/blockable_link';
 import FormattedMarkdownMessage from 'components/formatted_markdown_message';
-import noCompanyInfoGraphic from 'images/no_company_info_graphic.svg';
```
Comment (neallred) [inline components/admin_console/billing/company_info_display.tsx:11]: Nit: Can we remove this image? I don't see it used elsewhere in webapp.

File: components/admin_console/billing/payment_info_display.tsx [hunk 6]
```diff
@@ -7,12 +7,14 @@ import {useSelector} from 'react-redux';
 
 import {trackEvent} from 'actions/telemetry_actions';
 import BlockableLink from 'components/admin_console/blockable_link';
-import noPaymentInfoGraphic from 'images/no_payment_info_graphic.svg';
```
Comment (neallred) [inline components/admin_console/billing/payment_info_display.tsx:10]: Nit: can we remove this file? I don't see it used elsewhere in webapp.

**Review by neallred — APPROVED** [review]
Code looks good to me! There's some nonblocking comments on css classes & files I think it would be better to remove in this PR than later.

File: components/admin_console/billing/billing_subscriptions/billing_subscriptions.tsx [hunk 7]
```diff
@@ -164,7 +162,10 @@ export const contactSalesCard = (
                 }
             </div>
             <div className='PrivateCloudCard__image'>
-                <img src={isFreeTrial ? freeTrialPrivateCloudImage : privateCloudImage}/>
```
Comment (marianunez) [inline components/admin_console/billing/billing_subscriptions/billing_subscriptions.tsx:167]: Here we had two images, we may be missing the one for the case of free trial?

File: components/admin_console/billing/billing_summary/billing_summary.tsx [hunk 8]
```diff
@@ -10,19 +10,18 @@ import {BillingSchemes, CloudLinks, TrialPeriodDays} from 'utils/constants';
...
-import upgradeMattermostCloudImage from 'images/upgrade-mattermost-cloud-image.svg';
+import UpgradeSvg from 'components/common/svg_images_components/upgrade.svg';
+import ResetPasswordSvg from 'components/common/svg_images_components/reset_password.svg';
```
Comment (marianunez) [inline components/admin_console/billing/billing_summary/billing_summary.tsx:14]: Same as above regarding the `ResetPasswordSvg` name

**Review by marianunez — CHANGES_REQUESTED** [review]

File: components/admin_console/billing/billing_history.tsx [hunk 9]
```diff
@@ -15,7 +15,8 @@ import LoadingSpinner from 'components/widgets/loading/loading_spinner';
 import {pageVisited, trackEvent} from 'actions/telemetry_actions';
 import FormattedAdminHeader from 'components/widgets/admin_console/formatted_admin_header';
 import FormattedMarkdownMessage from 'components/formatted_markdown_message';
-import noBillingHistoryGraphic from 'images/no_billing_history_graphic.svg';
+import ResetPasswordSvg from 'components/common/svg_images_components/reset_password.svg';
```
[the remark this answers is not in the document]
Comment (pablovelezvidal) [reply components/admin_console/billing/billing_history.tsx:18]: I named it following the names that were in the figma design for each one of the SVGs. Maybe @matthewbirtch can help with a more intuitive name for this image?

File: components/admin_console/billing/billing_subscriptions/billing_subscriptions.tsx (hunk 7 above)
Comment (pablovelezvidal) [reply components/admin_console/billing/billing_subscriptions/billing_subscriptions.tsx:167]: I had understood that we would use the same image for both scenarios. Maybe @matthewbirtch can help us to confirm this.

File: components/admin_console/billing/billing_history.tsx (hunk 9 above)
[the remark this answers is not in the document]
Comment (marianunez) [reply components/admin_console/billing/billing_history.tsx:18]: Maybe it was reused for the email reset password designs (my guess), but if it's not reused in the webapp, let's name it something relevant for where we are using it i.e. `emptyBillingHistory`

[the remark this answers is not in the document]
Comment (matthewbirtch) [reply components/admin_console/billing/billing_history.tsx:18]: I think `reset password` was a mistake in the figma file. Apologies for that. I'm fine with `emptyBillingHistory`

File: components/admin_console/billing/billing_subscriptions/billing_subscriptions.tsx (hunk 7 above)
Comment (pablovelezvidal) [reply components/admin_console/billing/billing_subscriptions/billing_subscriptions.tsx:167]: Captura de pantalla 2021-08-11 a las 22 47 04

added the image and the condition back! @matthewbirtch @marianunez

**Comment by pablovelezvidal** [comment]
@marianunez @matthewbirtch can we defer the UX validation for this PR? since there is a problem with the spinwick server and there are a couple of other user stories related to changing images to the new branding, maybe we could UX validate these changes with another PR.

**Comment by pablovelezvidal** [comment]
> @pablovelezvidal since this a PR primarily about updating the UI, I don't see a need to rush the merge without UX review, do you?

nope besides that I would need some of these changes for the next user story I will work on, but that can be easily solved.

**Comment by matthewbirtch** [comment]
> nope besides that I would need some of these changes for the next user story I will work on, but that can be easily solved.

Ok, that's fine. If there is code you want to use for the next story, we can merge this and do a full UX review of the branding updates altogether (provided the Spinwick server issue is resolved)

**Review by matthewbirtch — APPROVED** [review]
Screenshots look good, but I'll need to do a closer inspection on the next PR with all illustrations as noted above

**Pull request merged**